.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -3,7 +3,7 @@ cross_MessageProcessingFrameworkTutorial:
   title: Use the &AWS; Message Processing Framework for .NET to publish and receive &SQS; messages
   title_abbrev: Use the &AWS; Message Processing Framework for .NET with &SQS;
   synopsis: create applications that publish and receive &SQS; messages using the &AWS; Message Processing Framework for .NET.
-  category: Cross-service examples
+  category: Cross
```

Comment:
This won't work right, because it'll create a new category named "Cross" and add a new section to the TOC.
The `category` field is used both as the literal string that appears in the TOC and the functional categorization enum.
This was probably a poor choice made for expedient reasons, and should maybe be revisited, but for this update I would update your code to use the full "Cross-service examples" and "Actions" for the two categories so that they exactly match other tools' expectations.

Reply:
Done